measure/src/main/scala/org/apache/griffin/measure/configuration/dqdefinition/DQConfig.scala

```diff
@@ -206,3 +211,23 @@ case class RuleOutputParam( @JsonProperty("type") private val outputType: String
 
   def validate(): Unit = {}
 }
+
+/**
+  * error configuration parameter
+  * @param columnName the name of the column
+  * @param errorType  the way to match error, regex or enumeration
+  * @param values     error value list
+  */
+@JsonInclude(Include.NON_NULL)
+case class RuleErrorConfParam( @JsonProperty("column.name") private val columnName: String,
+                               @JsonProperty("type") private val errorType: String,
+                               @JsonProperty("values") private val values: List[String]
+                             ) extends Param {
+  def getColumnName: Option[String] = if (StringUtils.isNotBlank(columnName)) Some(columnName) else None
+  def getErrorType: Option[String] = if (StringUtils.isNotBlank(errorType)) Some(errorType) else None
+  def getValues: Seq[String] = if (values != null) values else Nil
+
+  def validate(): Unit = {
+    assert("regex".equals(getErrorType.get) || "enumeration".equals(getErrorType.get), "error error.conf type")
```

Comment:
I think `equalsIgnoreCase` is better than `equals` here, just like in `CompletenessExpr2DQSteps.scala`.

Reply:
done